emission/pipeline/export_stage.py

```diff
@@ -55,7 +51,6 @@ def run_export_pipeline(process_number, uuid_list):
 
 
 def run_export_pipeline_for_user(uuid):
-    #run the intake stage?
```

Comment:
not sure what you mean by this comment?

Reply:
We resolved this in our last conversion, just removing the comment.